### ISSUE TITLE
Geometry.Explode Node Unexpected Behavior in 2.11 

### ISSUE THREAD

**Issue by karambk** [DynamoDS/Dynamo#11587]
## Dynamo version

2.11.0.4533 (Downloaded from dynamobim.org)

## Operating system

Windows 10

## What did you do? 

Subtracted two NurbsSurfaces with Surface.Difference node, then exploded the result surface

## What did you expect to see?

The exploded surface should stay the same as the result surface

## What did you see instead?

The biggest NurbsSurface of the initial input

Extra Notes:
Am not sure if this behavior is intended or not, but it broke a workflow I have previously.
I noticed that this started to happen only in 2.11 
You can check the attached file, just try it in 2.10 and in 2.11, you will directly be able to see the difference.
![compare](https://user-images.githubusercontent.com/74302224/113484132-633c0e80-94af-11eb-9448-df55349b071d.png)

[BUGdyn.zip](https://github.com/DynamoDS/Dynamo/files/6252961/BUGdyn.zip)

**Comment by aparajit-pratap** [comment]
@karambk thanks for reporting this. We could confirm that this is indeed a regression in 2.11. We have made a note of it and will fix it immediately. For the time being, is it necessary in your workflow to convert/use nurbs surfaces? If not, can you simply use the original generic surface and not convert/use NURBS. Thanks.

**Comment by karambk** [comment]
@aparajit-pratap thank you, after more testing, this problem seems to affect multiple other serious workflows that require surfaces join/cut operations..
In the meantime, don't worry about me I can manage some workarounds, but my main concern is for the version that will come with the next Revit 2022, this problem will cause some serious headaches if it is still here.. I really appreciate if this problem could be solved before the next Revit ships with it.

Thanks

**Comment by aparajit-pratap** [comment]
@karambk I agree with your assessment. We completely understand your concerns and will take the necessary steps to remediate this.
